.github/workflows/ci.yml

```diff
@@ -118,3 +118,49 @@ jobs:
           cd runtime-local/test
           pip install -r requirements.txt
           pytest -s ./integration/integration_test.py
+
+  run-k3d-integration-tests:
+    name: Run Integration Tests for runtime-k3d
+    runs-on: ubuntu-22.04
+    env:
+      DAPR_HOST_IP: 127.0.0.1
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Set commit id
+        run: |
+          NEW_CONFIG="$(jq --arg GITHUB_SHA "$GITHUB_SHA" '.packages[0].version |= $GITHUB_SHA' runtime-k3d/test/.velocitas.json)"
+          echo "${NEW_CONFIG}" > runtime-k3d/test/.velocitas.json
+
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Install required packages
+        run: |
+          sudo curl -L https://github.com/eclipse-velocitas/cli/releases/latest/download/velocitas-linux-x64 -o /usr/bin/velocitas
+          sudo chmod +x /usr/bin/velocitas
+          brew install k3d dapr/tap/dapr-cli
```

Comment:
Please pin the dapr version to the one we use in the template

Reply:
It is also fine to go back to the binaries if needed, since it is also way faster